src/sentry/static/sentry/app/views/performance/charts/index.tsx

```diff
@@ -17,13 +17,23 @@ import {getUtcToLocalDateObject} from 'app/utils/dates';
 import {IconWarning} from 'app/icons';
 import theme from 'app/utils/theme';
 
-import {ChartsContainer, ChartsGrid} from './styles';
+import {ChartsContainer, ChartsGrid} from '../styles';
 import Chart from './chart';
 import Footer from './footer';
 
 const YAXIS_OPTIONS = [
-  {label: 'Apdex', value: 'apdex(300)'},
-  {label: 'Throughput', value: 'rpm()'},
+  {
+    label: 'Apdex',
+    value: 'apdex(300)',
+    tooltip:
+      'Apdex is a ratio of satisfactory response times to unsatisfactory response times.',
+  },
+  {
+    label: 'Throughput',
+    value: 'rpm()',
+    tooltip:
+      'Throughput is the quantity of data being sent and received within a unit of time. ',
```

Comment:
You could also say 'RPM is the number of requests per minute that you are recording'

Reply:
ooOO thats better 🌮